README.md

```diff
@@ -23,3 +23,57 @@ The deployment pipeline is documented on the [sources repository for DevDocs](ht
 ## License
 
 Content from this documentation is licensed under the [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## Maintaining Hook list referential
+
+A complete referential of hook and informations on each hook is located in `modules/concepts/hooks/list-of-hooks/`. 
+
+This referential can be generated with a script, crawling all: 
```

Comment:
A script that we can put somewhere to the repo?

Reply:
as we spoke, it could be added as a Gist

Reply:
https://gist.github.com/thomasnares/b0605fdd9f5e77fa16bf5a42408feba3

I need to do some rewrite to fit the new format

Reply:
let's add a task to link `with a script` to this file once it's rewritten to the new format, ok?

Reply:
yes 👍